lighthouse-core/lib/lh-env.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * @license Copyright 2020 The Lighthouse Authors. All Rights Reserved.
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with the License. You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the specific language governing permissions and limitations under the License.
+ */
+'use strict';
+
+module.exports = {
+  // NODE_ENV is set to test by jest and by smokehouse CLI runner
+  // CI as a catchall for everything we do in Travis/GitHub Actions
+  isUnderTest: !!process.env.CI || process.env.NODE_ENV === 'test',
```

Comment:
> `!!process.env.CI`

I thought we wanted it to work in local testing too?

Reply:
We do and this does.

> NODE_ENV is set to test by jest and by smokehouse CLI runner

Do you not like the idea that there's possibly something in CI that we test we don't catch locally without `NODE_ENV`?

Reply:
> We do and this does.
> 
> >

oh, lol `||`. Ignore me :)